docs/src/content/docs/it/404.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Non trovato
+template: splash
+editUrl: false
+hero:
+  title: '404'
+  tagline: <strong>Houston, abbiamo un problema.</strong> Non siamo riusciti a trovare quella pagina.<br>Controlla l'URL o prova a utilizzare la barra di ricerca.
+  actions:
+    - text: Torna alla home
+      icon: right-arrow
+      link: /
```

Comment:
Thanks @itsmatteomanf!

Might not be as obvious the other links, but same applies to the 404 page here to avoid linking back to the English homepage.

```suggestion
      link: /it/
```